api/v1/submissions/PKPSubmissionHandler.php

```diff
@@ -550,6 +556,40 @@ public function delete($slimRequest, $response, $args)
         return $response->withJson($submissionProps, 200);
     }
 
+    /**
+     * Get the decisions associated to a submission
+     *
+     * @param Request $slimRequest Slim request object
+     * @param Response $response object
+     * @param array $args arguments
+     *
+     * @return Response
```

Comment:
I know the existing handlers use the `@param` and `@return` a lot, but let's try to start type-hinting in the function signature.

Reply:
Done.